### PR TITLE
Update versions after monthly release

### PR DESCRIPTION
**Pull request by backwind1233**

### CODE REVIEW

File: cosmos/azure-spring-boot-starter-cosmos/cosmos-multi-database-multi-account/pom.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-spring-data-cosmos</artifactId>
-      <version>3.12.0</version> <!-- {x-version-update;com.azure:azure-spring-data-cosmos;dependency} -->
+      <version>3.13.1</version> <!-- {x-version-update;com.azure:azure-spring-data-cosmos;dependency} -->
```
Comment (chenrujun) [inline cosmos/azure-spring-boot-starter-cosmos/cosmos-multi-database-multi-account/pom.xml:30]: Can we delete this, use bom instead?

File: cosmos/azure-spring-boot-starter-cosmos/cosmos-multi-database-single-account/pom.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-spring-data-cosmos</artifactId>
-      <version>3.12.0</version> <!-- {x-version-update;com.azure:azure-spring-data-cosmos;dependency} -->
+      <version>3.13.1</version> <!-- {x-version-update;com.azure:azure-spring-data-cosmos;dependency} -->
```
Comment (chenrujun) [inline cosmos/azure-spring-boot-starter-cosmos/cosmos-multi-database-single-account/pom.xml:25]: Same here.

File: spring-petclinic-microservices/pom.xml [hunk 3]
```diff
@@ -3,228 +3,228 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
+  <parent>
```
Comment (chenrujun) [inline spring-petclinic-microservices/pom.xml:6]: It's not easy to review this file.

File: spring-petclinic-microservices/pom.xml [hunk 4]
```diff
@@ -3,228 +3,228 @@
...
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.3.6.RELEASE</version>
```
Comment (chenrujun) [inline spring-petclinic-microservices/pom.xml:9]: It's better to use the same version of spring-boot-starter-parent

Comment (backwind1233) [reply spring-petclinic-microservices/pom.xml:9]: I agree, but we need to test the big sample, not just upgrade spring-boot-starter-parent.

File: spring-petclinic-microservices/pom.xml (hunk 3 above)
Comment (backwind1233) [reply spring-petclinic-microservices/pom.xml:6]: just reformat the pom, in intellj view it doesn't show the diff , but it shows on github.

File: cosmos/azure-spring-boot-starter-cosmos/cosmos-multi-database-single-account/pom.xml (hunk 2 above)
Comment (backwind1233) [reply cosmos/azure-spring-boot-starter-cosmos/cosmos-multi-database-single-account/pom.xml:25]: These two samples depend on azure-spring-data-cosmos directly,
in my concern, our bom is focus on starters,binders,etc. dependencies.

if we use bom instead, for these samples only, I don't it is necessary.

but for the entire azure-spring-boot-samples, I agree with you, so we can give our customer the same experience with other samples..

Comment (backwind1233) [reply cosmos/azure-spring-boot-starter-cosmos/cosmos-multi-database-single-account/pom.xml:25]: I think we can use another PR to address the issue, also we need to go through all samples to check other samples which can also import boms and test the samples .

**Review by chenrujun — APPROVED** [review]

**Pull request merged**